requirements.txt

```diff
@@ -0,0 +1,8 @@
+pandas==1.3.5
+opencv-python==4.6.0.66
+Pillow==9.2.0
+scikit-learn==1.0.2
+keybert==0.7.0
+sentence-transformers==2.2.2
+packages/ah_sidekick-0.8.7-py3-none-any.whl
```

Comment:
Version doesn't match package that is checked in. Please update version to 0.9.1

Reply:
Fixed - https://github.com/SalesHero-Inc/hero-document-clustering/pull/1/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R7